src/plugins/intel_cpu/src/nodes/conv.cpp

```diff
@@ -348,6 +348,8 @@ const std::vector<impl_desc_type>& Convolution::getDefaultImplPriority() {
         impl_desc_type::jit_avx512_dw,
         impl_desc_type::jit_avx512_1x1,
         impl_desc_type::jit_avx512,
+        impl_desc_type::brgconv_avx2_1x1,
```

Comment:
For now, we can't add brgconv_avx2 related into the list on avx2/avx2_vnni platform because of regression.  Maybe need to dynamic update the list based on platform ISA.

Reply:
how about erase brgconv related types from this priorities when isBrgConvAvailable is not true as the updated commit shows?